### ISSUE TITLE
Step RUN npm ci - error 

### ISSUE THREAD

**Issue by mulaneswa** [getumbrel/llama-gpt#76]
Hello i have an error when i try build container. Can someone Help ? 
Also i was add line to docker-compose.yml file ( - 'NODE_TLS_REJECT_UNAUTHORIZED=0'  ) didn't work 

[+] Building 93.9s (8/18)
 => [llama-gpt-ui internal] load build definition from Dockerfile                                                                                                                                     0.0s
 => => transferring dockerfile: 32B                                                                                                                                                                   0.0s
 => [llama-gpt-ui internal] load .dockerignore                                                                                                                                                        0.0s
 => => transferring context: 34B                                                                                                                                                                      0.0s
 => [llama-gpt-ui internal] load metadata for docker.io/library/node:19-alpine                                                                                                                       93.5s
 => CACHED [llama-gpt-ui] FROM ghcr.io/ufoscout/docker-compose-wait:latest                                                                                                                            0.4s
 => => resolve ghcr.io/ufoscout/docker-compose-wait:latest                                                                                                                                            0.4s
 => [llama-gpt-ui base 1/3] FROM docker.io/library/node:19-alpine@sha256:8ec543d4795e2e85af924a24f8acb039792ae9fe8a42ad5b4bf4c277ab34b62e                                                             0.0s
 => [llama-gpt-ui internal] load build context                                                                                                                                                        0.1s
 => => transferring context: 13.73kB                                                                                                                                                                  0.1s
 => CACHED [llama-gpt-ui base 2/3] WORKDIR /app                                                                                                                                                       0.0s
 => CACHED [llama-gpt-ui base 3/3] COPY package*.json ./                                                                                                                                              0.0s
 => ERROR [llama-gpt-ui dependencies 1/1] RUN npm ci                                                                                                                                                 92.5s
------
 > [llama-gpt-ui dependencies 1/1] RUN npm ci:
92.13 npm ERR! code EPROTO
92.14 npm ERR! syscall write
92.14 npm ERR! errno EPROTO
92.15 npm ERR! request to https://registry.npmjs.org/zwitch/-/zwitch-2.0.4.tgz failed, reason: write EPROTO 00BDA786FFFF0000:error:0A000152:SSL routines:final_renegotiate:unsafe legacy renegotiation disabled:../deps/openssl/openssl/ssl/statem/extensions.c:921:
92.15 npm ERR!
92.26
92.26 npm ERR! A complete log of this run can be found in: /root/.npm/_logs/2023-08-28T08_54_52_444Z-debug-0.log
------
failed to solve: executor failed running [/bin/sh -c npm ci]: exit code: 1

**Comment by bastiedotorg** [comment]
TL;DR reboot
Hi, 
I assume you did the same as me: install docker without restarting your machine. It seems that the iptables configuration was screwed.
A reboot of the linux machine helped and it worked.
